charts/kong-operator/templates/services.yaml

```diff
@@ -31,3 +31,20 @@ spec:
   selector:
     app.kubernetes.io/component: ko
 {{- end -}}
+{{- if .Values.enableConversionWebhook -}}
```

Comment:
We're checking here `.Values.enableConversionWebhook` but above (in `charts/kong-operator/templates/deployment.yaml`) we're checking `.Values.conversionWebhook.enabled`. Shouldn't this check the same thing for the condition?